docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeLogMeanVarianceFixZero.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.NormalizingTransformer;
+
+namespace Samples.Dynamic
+{
+    public class NormalizeLogMeanVarianceFixZero
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging,
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+            var samples = new List<DataPoint>()
+            {
+                new DataPoint(){ Features = new float[5] { 1, 1, 3, 0, float.MaxValue } },
+                new DataPoint(){ Features = new float[5] { 2, 2, 2, 0, float.MinValue } },
+                new DataPoint(){ Features = new float[5] { 0, 0, 1, 0, 0} },
+                new DataPoint(){ Features = new float[5] {-1,-1,-1, 1, 1} }
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+            // NormalizeLogMeanVariance normalizes the data based on the computed mean and variance of the logarithm of the data.
+            // Uses Cumulative distribution function as output.
+            var normalize = mlContext.Transforms.NormalizeLogMeanVariance(true, "Features", useCdf: true);
+
+            // NormalizeLogMeanVariance normalizes the data based on the computed mean and variance of the logarithm of the data.
```

Comment:
`NormalizeLogMeanVariance normalizes the data based on the computed mean and variance of the logarithm of the data.` doesn't tell much more than the line below. We need some equations here.

Reply:
Yes that would be nice I can add that here. 

---
In reply to: [298254677](https://github.com/dotnet/machinelearning/pull/3916#discussion_r298254677) [](ancestors = 298254677)

Reply:
As I told you the other day I am still concerned about that equation. I am going to do a separate PR where I add the equations. (Same for the documentation comment in the catalog that you had).
There is no documentation on how to derive them so I am going through the math again and trying to understand the rational.

---
In reply to: [298311132](https://github.com/dotnet/machinelearning/pull/3916#discussion_r298311132) [](ancestors = 298311132,298254677)